From 8350cae4f195b47afb4e5f3b3f5bed1dd40767ac Mon Sep 17 00:00:00 2001
From: Craig Small <csmall@enc.com.au>
Date: Wed, 3 Jun 2015 21:18:27 +1000
Subject: [PATCH 19/21] doc: fix typo in kill1ll.1

Lev reported that we were saying in the manpage that killall uses
expended regular expression and not exTended regular expression.
---
 doc/killall.1 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/killall.1 b/doc/killall.1
index afa8ed7..4e9c301 100644
--- a/doc/killall.1
+++ b/doc/killall.1
@@ -1,12 +1,12 @@
 .\"
 .\" Copyright 1993-2002 Werner Almesberger
-.\"           2002-2012 Craig Small
+.\"           2002-2015 Craig Small
 .\" This program is free software; you can redistribute it and/or modify
 .\" it under the terms of the GNU General Public License as published by
 .\" the Free Software Foundation; either version 2 of the License, or
 .\" (at your option) any later version.
 .\"
-.TH KILLALL 1 "2012-7-28" "psmisc" "User Commands"
+.TH KILLALL 1 "2015-07-03" "psmisc" "User Commands"
 .SH NAME
 killall \- kill processes by name
 .SH SYNOPSIS
@@ -132,7 +132,7 @@ are s,m,h,d,w,M,y for seconds, minutes, hours, days, weeks, Months and
 years respectively.
 .IP "\fB\-Z\fP, \fB\-\-context\fP"
 (SELinux Only) Specify security context: kill only processes having
-security context that match with given expended regular expression
+security context that match with given extended regular expression
 pattern.  Must precede other arguments on the command line.  Command
 names are optional.
 .SH FILES
-- 
2.5.2

